central/pruning/pruning_test.go

```diff
@@ -586,6 +604,70 @@ func (s *PruningTestSuite) TestImagePruning() {
 	}
 }
 
+func (s *PruningTestSuite) TestReportHistoryPruning() {
+	s.T().Setenv(features.VulnMgmtReportingEnhancements.EnvVar(), "true")
```

Comment:
Skip test if the feature flag can't be set. This is because only debug builds allow setting the feature flag.
```
s.T().Setenv(features.VulnMgmtReportingEnhancements.EnvVar(), "true")
if !features.VulnMgmtReportingEnhancements.Enabled() {
		s.T().Skip("Skip tests when ROX_VULN_MGMT_REPORTING_ENHANCEMENTS disabled")
		s.T().SkipNow()
	}
```